cloudcannon.config.yml

```diff
@@ -0,0 +1,174 @@
+paths:
+  data: data
+  collections: content
+  includes: layouts
+  layouts: layouts
+  static: static
+  uploads: static/uploads
+  source: ''
+  archetypes: archetypes
+  assets: assets
+  content: content
+  pages: content
+  publish: public
+  config: ''
+collections_config:
+  pages:
+    path: content
```

Comment:
`content` folder does not exist. In this project its called `pages`. Either replaces `content` with `pages` or rename the folder